test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -79,9 +79,11 @@ test.requestHooks(requestLogger, EnrollProfileSignUpWithAdditionalFieldsMock)('s
   await identityPage.clickSignUpLink();
 
   requestLogger.clear();
+  await t.expect(await enrollProfilePage.getFormFieldLabel('userProfile.country')).eql('Country');
```

Comment:
also add a test that checks for `textbox` as well ?

Reply:
test added. Thanks